src/components/seller/ProductRegistration.tsx

```
@@ -104,33 +117,324 @@ export function ProductRegistration() {
+      if (formData.categorySub) {
+        const main = categories.find((c) => c.label === formData.categoryMain);
+        const middle = main?.children?.find((c) => c.label === formData.categoryMiddle);
+        if (middle && middle.children) {
+          const id = findCategoryId(formData.categorySub, middle.children);
+          if (id) categoryIds.push(id);
+        }
+      }
+      // 태그 ID 배열
+      const tagCategoryIds = selectedTags.map((tag) => tag.value);
+      // 옵션
+      const productOptions = optionArray.items.map((opt) => ({
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)